crates/sncast/src/lib.rs

```diff
@@ -86,6 +88,20 @@ pub fn get_provider(url: &str) -> Result<JsonRpcClient<HttpTransport>> {
     Ok(provider)
 }
 
+pub async fn warn_if_incompatible_rpc_version(
+    provider: &JsonRpcClient<HttpTransport>,
+    url: &str,
+) -> Result<()> {
```

Comment:
I think this function could be moved to shared crate and used both in sncast and snforge

Reply:
Does it make sense to create `mod print` int the `shared` crate and move `print_as_warning` there, so that it can be used in both forge and cast?